model-optimizer/extensions/middle/UpsampleToResample.py

```diff
@@ -79,15 +79,6 @@ def replace_pattern(self, graph: Graph, match: Dict[str, Node]):
             height_scale = upsample['height_scale']
             width_scale = upsample['width_scale']
 
-        if not math.isclose(height_scale, width_scale, rel_tol=1e-5):
-            log.debug('Width and height scales are not equal: {} vs {} for node {}'.format(
-                width_scale, height_scale, upsample_name))
-            return
-        if depth_scale is not None and not math.isclose(height_scale, depth_scale, rel_tol=1e-5):
```

Comment:
Such changes there are also in PR https://github.com/openvinotoolkit/openvino/pull/970.

Reply:
And in the above mentioned PR Interpolate-4 is generated and tests are corrected according to the last variant of Interpolate-4 specification.